src/components/TransactionFailText/index.tsx

```diff
@@ -6,11 +6,18 @@ import Img from 'src/components/layout/Img'
 import InfoIcon from 'src/assets/icons/info_red.svg'
 
 import { useSelector } from 'react-redux'
-import { currentSafeThreshold } from 'src/logic/safe/store/selectors'
 import { shouldSwitchWalletChain } from 'src/logic/wallets/store/selectors'
 import { grantedSelector } from 'src/routes/safe/container/selector'
 import { EstimationStatus } from 'src/logic/hooks/useEstimateTransactionGas'
 
+enum ErroMessage {
+  general = 'This transaction will most likely fail.',
+  creation = 'To save gas costs, avoid creating the transaction.',
+  execution = 'To save gas costs, reject this transaction.',
+  notOwner = `You are currently not an owner of this Safe and won't be able to submit this transaction.`,
```

Comment:
You can remove the backticks

Reply:
It's for the apostrophe in `won't`.

Reply:
My bad, I normal use double quotation marks in that instance. Might start using this style from now on myself.